packaging/windows/WINDOWS_INSTALLER.md

```diff
@@ -54,11 +54,17 @@ This section provides instructions for installing Netdata in silent mode, which
 Connect your Agent to your Netdata Cloud Space with token `<YOUR_TOKEN>` and room `<YOUR_ROOM>`:
 
 ```bash
-netdata-installer.exe /S /A /TOKEN=<YOUR_TOKEN> /ROOMS=<YOUR_ROOM>
+netdata-installer-x64.exe /S /A /TOKEN=<YOUR_TOKEN> /ROOMS=<YOUR_ROOM>
 ```
 
 Replace `<YOUR_TOKEN>` and `<YOUR_ROOM>` with your actual Netdata Cloud Space claim token and room ID, respectively.
 
+> **Note**
+>
+> The Windows version of Netdata is intended for users on paid plans.
+>
+> The Windows installer is currently under beta, and thus it is only available in the nightly release channel. A stable version will be released soon.
```

Comment:
Why put it here? Please move it to the beginning of the doc.

Reply:
at the beginning of the doc you **already have**

> This feature is currently only available for Nightly releases, and the installer can be found in our nightlies repo

Reply:
I removed bit about the nightly